Library/Homebrew/rubocops/lines.rb

```diff
@@ -455,6 +455,149 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
         end
       end
 
+      # This cop makes sure that the `generate_completions_from_executable` DSL is used.
+      #
+      # @api private
+      class GenerateCompletionsDSL < FormulaCop
+        extend AutoCorrector
+
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          install = find_method_def(body_node, :install)
```

Comment:
This can be `nil` if there's no install method (which could happen with abstract formulae as we do in `Homebrew/homebrew-portable-ruby`. That seems to be what's causing the problem. You can fix this by adding

```suggestion
          install = find_method_def(body_node, :install)
          return if install.blank?
```

Reply:
Thanks, that's what I was missing